src/modules/cmdpal/extensionsdk/nuget/BuildSDKHelper.ps1

```diff
@@ -0,0 +1,102 @@
+Param(
+  [string]$Configuration = "release",
+  [string]$VersionOfSDK = "0.0.0",
```

Comment:
should this default to 0.0.1 too? 🤷

Reply:
This will get overridden by:
1. pipeline: Version number set in the yaml
2. local builds: Passed in version number

There are two reasons to keep this as 0.0.0:
1. It makes it very evident if something went wrong during builds
2. We don't want to have to worry about updating this number in multiple locations every time we rev the version number.